docs/src/tutorial/simple_system.md

```diff
@@ -122,6 +160,12 @@ If you are not sure whether you have the latest version, please upgrade to ensur
     *Add relationships* from the context menu. The *Add relationships*
     dialog will pop up.
 
+!!! note
+    Alternatively right click the objects in the graph view and 
+    *add relationships* will show the available relationships. 
+    If your relationship relies on nodes and units they both need 
```

Comment:
This is something I found confusing for the first run-through. I kept creating things and then they'd vanish! Maybe note somewhere that things have to be selected to show up? Or tell them how to make everything visible earlier on?

Reply:
I don't know exactly what happened to you but I've rephrased it and I think it should be clearer now.